src/api/server.js

```diff
@@ -0,0 +1,118 @@
+const express = require('express');
```

Comment:
While this file is useful and should definitely be versioned, it should not be kept in `src` directory becuase it is for local development; in production we will have a different Mongo instance.